todo.go

```diff
@@ -128,9 +128,22 @@ func routeInput(command string, input string) {
 	case "init":
 		app.InitializeRepo()
 	case "web":
-		web := todolist.NewWebapp()
-		fmt.Println("Now serving todolist web.\nHead to http://localhost:7890 to see your todo list!")
-		open.Start("http://localhost:7890")
-		web.Run()
+		// Only open default browser if .todos.json exists
+		pwd, err := os.Getwd()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if _, err := os.Stat(pwd + "/.todos.json"); err == nil {
```

Comment:
You can get this functionality by instantiating a FileStore and running [Load](https://github.com/gammons/todolist/blob/master/todolist/file_store.go#L19) on it.  That will cause the program to exit if there is not a .todos.json in the cwd.  Then you're safe to start the webapp after that.